packages/formation-react/src/components/Table/Table.jsx

```diff
@@ -48,19 +49,26 @@ function Table(props) {
             className={`${borderClasses} ${rowPaddingClass}`}
             role="row"
           >
-            {fields.map((field, index) => (
-              <td
-                data-index={index}
-                className={classNames(borderClasses, {
-                  'vads-u-text-align--left': field.alignLeft,
```

Comment:
Could this be accomplished by adding another entry here?

```
'medium-screen:vads-u-text-align--right': field.alignRight
```

Reply:
@bkjohnson Yep it can, I figured putting it in an if statement would be easier to read but thinking about it I suppose it is really not. I updated this.